cloudsync/oauth/oauth_config.py

```diff
@@ -118,6 +118,9 @@ def wait_auth(self, token_url, timeout=None, **kwargs):
                     raise OAuthError(self.failure_info)
                 raise OAuthError("Oauth interrupted")
 
+            if "timeout" not in kwargs:
```

Comment:
why not set the timeout default to 60?

Reply:
I guess someone could explicitly set it to None

Reply:
not sure I follow... can you even have a named param and kw param with the same name? and are these meant to be different timeouts? if so, can they have different names?

Reply:
thats what this does

Reply:
ok i changed it to  be more clear.  the named param is a real timeout.   the timeout passed to the requests lib is an interpacket timeout that has no bearing on the real world notion of timeouts.